tsconfig.json

```diff
@@ -8,11 +8,13 @@
     ],
     "sourceMap": true,
     "rootDir": "src",
-    "strict": true   /* enable all strict type-checking options */
+    "strict": true,   /* enable all strict type-checking options */
     /* Additional Checks */
     // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
     // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
     // "noUnusedParameters": true,  /* Report errors on unused parameters. */
+    "resolveJsonModule" : true,
+    "esModuleInterop": true
```

Comment:
I think its better to move after `"strict": true,` line.

Reply:
I revised it. Thank you.